fe-nc-news-react/src/components/ArticleCard.jsx

```diff
@@ -1,14 +1,20 @@
 
 export default function ArticleCard({ article }) {
     return (
-      <li key={article.id} className="article-card">
-        <h2>{article.title}</h2>
-        <p>{article.topic}</p>
-        <p>{article.author}</p>
+      <li className="article-card">
+        <img
+          className="article-img"
+          src={article.article_img_url}
+          alt="article-image"
+        />
+        <div className="article-info">
+        <h2 className="article-card-header">{article.title}</h2>
+        <p>Author: {article.author}</p>
         <p>{article.created_at}</p>
```

Comment:
We also want to see when the articles were created in a human readable format as they should already be sorted by date when they come back from the api